### ISSUE TITLE
make 'real' sectors more realistic

### ISSUE THREAD

**Issue by porcuquine** [filecoin-project/rust-fil-proofs#467]
We need to start using realistic parameters for sectors, and `go-filecoin` needs to start using those sectors (the `go-filecoin` side should have its own story once this work is done). This is going to mean much larger proof sizes and proving times than the `ProofTest` config that has been in use in the dev cluster.

Since there are some complexities to sort through with the expanded challenge counts, along with interactions between challenge tapering and partitions, the first pass at this may still be partial.

The goal is to find a set of parameters, including sector size, which is usable for devnet and matches our current understanding of the security requirements.
